dspace/config/modules/info-actuator.cfg

```diff
@@ -0,0 +1,19 @@
+
+info.app.name = ${dspace.name}
+info.app.version = 7.3
+
+info.app.dir = ${dspace.dir}
+info.app.url = ${dspace.server.url}
+info.app.db = ${db.url}
+info.app.solr.server = ${solr.server}
+info.app.solr.prefix = ${solr.multicorePrefix}
+info.app.mail.server = ${mail.server}
+info.app.mail.from-address = ${mail.from.address}
+info.app.mail.feedback-recipient = ${feedback.recipient}
+info.app.mail.mail-admin = ${mail.admin}
+info.app.mail.mail-helpdesk = ${mail.helpdesk}
+info.app.mail.alert-recipient = ${alert.recipient}
```

Comment:
We should add some basic inline comments & docs here to let people know that you can remove one of these settings from being reported from the `/actuator/info` endpoint by just commenting it out & that you can add new settings by just creating a new key starting with "info.app".